### PR TITLE
Updated alertmanagerconfig sample to use slack as example

### PR DESCRIPTION
**Pull request by talha0324**

### CODE REVIEW

File: content/sre/alerting/workload-application-alerts.md [hunk 1]
```diff
@@ -45,28 +45,52 @@ A sample AlertmanagerConfig can be configured in [Application Chart](https://git
...
-       groupInterval: 5m
-       repeatInterval: 12h
-    receivers: 
-      - name: "null"
+      receiver: 'slacknotifications'
+    receivers:
+    - name: 'slacknotifications'
+      slackConfigs:
+      - apiURL: 
+          name: slackconfig-test
```
Comment (rasheedamir) [inline content/sre/alerting/workload-application-alerts.md]: poor name of choice @talha0324

File: content/sre/alerting/workload-application-alerts.md [hunk 2]
```diff
@@ -45,28 +45,52 @@ A sample AlertmanagerConfig can be configured in [Application Chart](https://git
...
+            insecureSkipVerify: true
+```
+
+Above slack config is pulled from a secret present in the same namespace, which should look like this:
+```
+kind: Secret
+apiVersion: v1
+metadata:
+  name: slackconfig-test
+  namespace: alert-test
```
Comment (rasheedamir) [inline content/sre/alerting/workload-application-alerts.md]: create a new namespace with this name?

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-193-58a14472`

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-193-6f195fec`

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-193-1abfc31d`

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-193-cdd59e52`

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-193-07e34648`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-193-87c37ba0`

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.134`

**Pull request merged**